src/Bundle/Resources/config/services/controller.xml

```diff
@@ -19,40 +19,69 @@
             <argument type="service" id="service_container" />
             <argument type="service" id="sylius.expression_language" />
         </service>
+        <service id="Sylius\Bundle\ResourceBundle\Controller\ParametersParserInterface" alias="sylius.resource_controller.parameters_parser" public="false" />
```

Comment:
Can we reverse this definition? So the interface will be on a service itself and `sylius.resource_controller.parameters_parser` will alias to it? Same as requested: https://github.com/Sylius/SyliusGridBundle/pull/279#discussion_r1021603167